### PR TITLE
Remove urllib import (Python3 fix)

### PR DESCRIPTION
**Pull request by vthorsteinsson**
`import urllib`

does not work in Python3. Instead,

`import six.moves.urllib_request`

imports `urllib` on Python2 and `urllib.request` on Python3.

### CODE REVIEW

**Comment by ilya-chistyakov** [comment]
Maybe `import six.moves.urllib_request as urllib` will be more correct.

File: tensor2tensor/data_generators/generator_utils.py [hunk 1]
```diff
@@ -22,13 +22,12 @@
...
 import os
 import tarfile
-import urllib
 
 # Dependency imports
 
 import six
 from six.moves import xrange  # pylint: disable=redefined-builtin
-import six.moves.urllib_request
+import six.moves.urllib_request # Imports urllib on Python2, urllib.request on Python3
```
Comment (lukaszkaiser) [inline tensor2tensor/data_generators/generator_utils.py]: This doesn't seem to work for us with python2 at Google. Could you try this:
import six.moves.urllib_request as urllib

**Comment by vthorsteinsson** [comment]
Fixed with thanks to @ilya-chistyakov. Sorry for the mixup. It works now in my tests both on Python2 and Python3.

**Comment by lukaszkaiser** [comment]
Thanks, that works good.

**Pull request merged**